operations/network/management_network/aruba/requirements_and_optional__configuration.md

```diff
@@ -0,0 +1,22 @@
+# How to connect management network to your campus network
```

Comment:
The new file name contains a excess underscore (requirements_and_optional__configuration.md), so it is treated as a new file, instead of renaming.

Reply:
Not sure why these names got changed in this PR. Fixed and removed duplicate.